app/routes/bestuurseenheid/zitting/notulen.js

```diff
@@ -1,18 +1,26 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingNotulenRoute extends Route {
-  breadCrumb = { title: 'Notulen' };
+  @service store;
 
   async model() {
-    const meeting = this.modelFor('bestuurseenheid.zitting');
-    const meetingNotes = await this.store.query('notulen', {
-      "filter[zitting][:id:]": meeting.id,
-      sort: "-publication.created-on",
-      include: "publication"
-    });
-
-    const notulen = meetingNotes.firstObject;
-    return { notulen, zitting: meeting };
+    //Why was it done this way:
+    //const meeting = await this.modelFor('bestuurseenheid.zitting');
+    //const meetingNotes = await this.store.query('notulen', {
+    //  'filter[zitting][:id:]': meeting.id,
+    //  sort: '-publication.created-on',
+    //  include: 'publication',
+    //});
+    //const notulen = meetingNotes.firstObject;
+    //return { notulen, zitting: meeting };
+    //When this looks easier and works the same:  ?
+    const zitting = await this.modelFor('bestuurseenheid.zitting');
+    const notulen = await zitting.notulen;
+    await notulen.publication;
+    const bestuursorgaan = await zitting.bestuursorgaan;
+    await bestuursorgaan.isTijdsspecialisatieVan;
+    return zitting;
+    //Could have used fastboot.deferRendering, but even in normal mode, we want to wait untill all this data is loaded before rendering the page, so that nothing jumps on screen.
```

Comment:
I would delete the comments in this file. Things like this should just be discussed as a comment in the PR imo.